### ISSUE TITLE
GitHub Project Migration (Active Project Board)

### ISSUE THREAD

**Issue by JackHaeg** [hackforla/VRMS#1661]
### Overview
We need to prepare the Project Boards for the forced migration that GitHub is doing in August, so that it is already working well for the team by then.

### Action Items
- [x]  copy this template to a comment below
   ````
   ### Text from project board card
   ```
   [INSERT MARKDOWN HERE]
   ```
   - [ ] Determine where this item goes on the wiki (page and section)
   - [ ] copy item to wiki
   - [ ] hide this comment when completed.
   ````
- [x] copy markdown from a card in the "Start Here" column of the Project Board into your new comment where it says `[INSERT MARKDOWN HERE]
- [x] Follow this approach for all cards on this active project board (see "Resources/Instructions" for filtered project board link)
- [x] Jack - review & delete cards from project board.
- [x] Jack - add relevant card info to the wiki or to the column details on the new project board wherever relevant
- [x] Jack - delete old board, set up workflows. 
- [x] Jack - close issue.

### Resources/Instructions
- Project board URL (filtered by card only): https://github.com/hackforla/VRMS/projects/12?card_filter_query=is%3Anote
- New Project Board: https://github.com/orgs/hackforla/projects/72/views/1

**Comment by JackHaeg** [comment]
### Text from project board card, General Information/Resources
```
Team Meeting Times:

All Team
- Monday 7:00 - 8:00 pm PST

Planning (PMs / Design)
- Thursdays 4:00 - 5:00 pm PST

```
- [x] Determine where this item goes on the wiki (page and section)
- [x] copy item to wiki
- [x] hide this comment when completed.

Team Meeting Times are stored on the Wiki on the following page: [How We Run Team Meetings](https://github.com/hackforla/VRMS/wiki/How-We-Run-Team-Meetings)

**Comment by JackHaeg** [comment]
### Text from project board card, from VRMS v0.3 Development ("Start Here" column)
```
Quick Links
- [Strategy and Vision](https://docs.google.com/presentation/d/1DLljKCoaoI2JSZUxL6oIOhBBXbSF30uW0EOgtYNSRrs/edit#slide=id.p)
- [Roadmap and v0.4 (MVP) Release Plan](https://docs.google.com/document/d/1BlhtbYBbSBlwSKNFupqUwdtKiUluCErIt0Mr4vNzxlA/edit#heading=h.2q2gemodqthk)
- [UI Mockups (v0.4)](https://www.figma.com/file/9c6pxjImN8MjHAZLH7zxYm/?node-id=24553%3A2603)
- [UI Mockups (v1.0)](https://www.figma.com/file/9c6pxjImN8MjHAZLH7zxYm/VRMS-Mocks?node-id=23726%3A2254)
- [MVP/0.4 Requirements](https://docs.google.com/document/d/16VvC0fbaXsvEQkQaVSj8dG3wFEHgErbjJ6H3akSUoVM/edit?usp=sharing)
- [Database Schema - out of date](https://drive.google.com/file/d/1sSqbKyEHE9g5zn_rJ6ONUCW2TPVkTYKf/view?usp=sharing)
- [Dev 0.3](http://18.117.69.247/)
```
- [ ] Determine where this item goes on the wiki (page and section)
- [ ] copy item to wiki
- [ ] hide this comment when completed.

**Comment by Steph112792** [comment]
Issues identified that are going to get addressed, but are not being started because of dependencies. Make sure dependencies are clearly identified!

**Comment by JackHaeg** [comment]
### Text from project board card, Icebox
```
Issues identified that are going to get addressed, but are not being started because of dependencies. Make sure dependencies are clearly identified!
```
- [x] copy text to new project board column details.
- [x] hide this comment when completed.

**Comment by Steph112792** [comment]
### Text from project board card, New Issue Approval
```
All issues that are newly created or coming out of the icebox need to be reviewed by Product to make sure they are complete and prioritized.  Make sure all issues are labeled with 1) Role, 2) Size, 3) Feature
```
- [x] copy text to new project board column details.
- [x] hide this comment when completed.

**Comment by Steph112792** [comment]
### Text from project board card, Prioritized Backlog
```
Issues that are ready to be worked on. This column is prioritized by most important first!
```
- [x] copy text to new project board column details.
- [x] hide this comment when completed.

**Comment by Steph112792** [comment]
### Text from project board card, In Progress
```
Tasks/issues actively being worked and has been assigned to team member. WIP Limit: 1 issue = 1 person
```
- [x] copy text to new project board column details.
- [x] hide this comment when completed.

**Comment by Steph112792** [comment]
### Text from project board card, Epic Issues
```
Epic issues that are currently in progress.
```
- [x] copy text to new project board column details.
- [x] hide this comment when completed.

**Comment by Steph112792** [comment]
### Text from project board card, Questions/Review
```
Issues that you are blocked on because you need an answer from another team member of other type of question/review.
```
- [x] copy text to new project board column details.
- [x] hide this comment when completed.

**Comment by Steph112792** [comment]
### Text from project board card, General Information/Resources
```
V4 Product Spec - https://docs.google.com/document/d/16VvC0fbaXsvEQkQaVSj8dG3wFEHgErbjJ6H3akSUoVM/edit#
```
- [x] Determine where this item goes on the wiki (page and section)
- [x] copy item to wiki: https://github.com/hackforla/VRMS/wiki/GitHub-Migration-%E2%80%90-2024.06.20
- [x] hide this comment when completed.

**Comment by Steph112792** [comment]
### Text from project board card, Epic Issues
```
Epic issues in waiting state (e.g., drafts, waiting for changes, etc.)
```
- [x] Determine where this item goes on the wiki (page and section) - _BOARD ONLY, NOT COPYING TO NEW BOARD, AS ANY DRAFT EPIC WILL BE PLACED IN THE "NEW ISSUE APPROVAL" OR "ICEBOX" IF PAUSED BY DEPENDENCY._
- [x] copy item to wiki - _WE WILL NOT BE COPYING THIS OVER._
- [x] hide this comment when completed.

**Comment by JackHaeg** [comment]
### Text from project board card, Done
```
Great work! Time to celebrate the little wins.
..................       <("<) <( " )> (>")>   .................
```
- [x] copy text to new project board column details.
- [x] hide this comment when completed.

**Comment by JackHaeg** [comment]
Migration of active board is complete.
